webpage/_data/navigation_v2.yml

```diff
@@ -0,0 +1,38 @@
+# Main links links.
+main:
+  - title: "About"
+    url: /about
+  - title: "Gallery"
+    url: /
+  - title: "Installing the IOOS conda environment"
+    url: /conda
+
+# Site navigation links.
+site_nav:
+  - title: Video Tutorials
+    children:
+      - title: "*Python tools*"
+      - title: "* Catalog driven reproducible workflows for ocean-science"

```

Comment:
@jbosch-noaa as I suspect this will not render as expected. Even though the `*` is the proper way to get bullets in markdown the navigation section is created following the theme rules and the theme must be changed in order to achieve that. (Ping @mwengren here in case he knows a workaround.)